src/test/java/stepDefinitions/android/PostsLoadWhenScrollingDownVertical.java

```diff
@@ -40,15 +41,15 @@ public void iScrollDownToTheLastPostInTheFirstPayload(int totalPosts) throws Thr
 
         while (verticalPostTitles.size() < totalPosts) {
             int totalTitlesBeforeSwipe = verticalPostTitles.size();
-            wait.until(new ElementsPresent(postHeadline));
+            wait.until(new ElementsPresent(postHeadlines));
             verticalPostTitles.addAll(getVerticalPostTitles());
             verticalSwipe(0.90, 0.30);
             wait.until(new ElementNotVisible(spinner));
             int totalTitlesAfterSwipe = verticalPostTitles.size();
 
             Boolean areMorePostsPresent = totalTitlesAfterSwipe > totalTitlesBeforeSwipe;
             // Verify that more posts are displayed
-            Assert.assertTrue(areMorePostsPresent, "ERROR: MORE POSTS DID NOT LOAD");
+            Window.assertTrue(areMorePostsPresent, "ERROR: MORE POSTS DID NOT LOAD");
```

Comment:
Using the Window.assertTrue method catches the exception, but the test still pass